people_velocity_tracker/scripts/tracker.py

```diff
@@ -91,9 +91,14 @@ def __init__(self):
         self.people = {}
         self.TIMEOUT = rospy.Duration(rospy.get_param('~timeout', 1.0))
         self.sub = rospy.Subscriber('/people_tracker_measurements',
-                                    PositionMeasurementArray, self.pm_cb)
-        self.mpub = rospy.Publisher('/visualization_marker', Marker)
-        self.ppub = rospy.Publisher('/people', People)
+                                    PositionMeasurementArray,
+                                    self.pm_cb)
+        self.mpub = rospy.Publisher('/visualization_marker',
+                                    Marker,
+                                    queue_size=0)
```

Comment:
You are correct! I had meant to call out the default behavior was being executed (e.g. Queuesize=None) I mistook zero and None to have the same behavior... 

Sorry for the mistake! I'll fix it.